examples/basics/datasets.ipynb

```diff
@@ -1,18 +1,16 @@
 {
```

Comment:
<p>Could you remove this dataset id, and replace it with a place holder string &lt;DATASET_ID&gt;?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/1081/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>